### ISSUE TITLE
Base table or view not found: 1146 Table 'vendor_name.razorpay_sales_order' 

### ISSUE THREAD

**Issue by ashok1421** [razorpay/razorpay-magento#481]
Hello @yashgit891 ,
I found the issue on the Magento backend. 

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'vendor_name.razorpay_sales_order' doesn't exist, query was: DESCRIBE `razorpay_sales_order`
Exception in /data/www/vendor_name/vendor/magento/framework/DB/Adapter/Pdo/Mysql.php:597
```

**Comment by ashok1421** [comment]
Please consider this pull request for Solutions fo this problem
https://github.com/razorpay/razorpay-magento/pull/482

**Comment by yashgit891** [comment]
@ashok1421 when exactly was this error occurring?

**Comment by ashok1421** [comment]
@yashgit891 admin side all page

**Comment by yashgit891** [comment]
@ashok1421 not able to reproduce this issue, tried on multiple systems. Which log file is it getting logged in? Also can you check on some other setup if the same is occurring?

**Comment by yashgit891** [comment]
@ashok1421 was able to replicate the issue and found that your PR fixes the problem, could not merge it due to some workflows not running correctly. Issue should be resolved in latest release 4.1.6. Marking this closed now.
